### PR TITLE
Business contracts definition merge to master

### LINKED ISSUE #8: Business contracts definition

### PR DESCRIPTION
**Pull request by vvillena**
- Definition of interfaces
- Creation of ILinkAssigner and ILinkFinder
- Creation of models

#8

### CODE REVIEW

**Review by xaberue — DISMISSED** [review]

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkAssigner.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ChustaSoft.GamersPlatformUtils.Abstractions.Contracts
+{
+    public interface ILinkAssigner
+    {
+        Task<bool> Assign(IEnumerable<GameLink> gameLinks);
```
Comment (Arthus15) [inline ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkAssigner.cs]: Async sufix

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkFinder.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ChustaSoft.GamersPlatformUtils.Abstractions.Contracts
+{
+    public interface ILinkFinder
+    {
+        string RootFolderName { get; set; }
+
+        Task<IEnumerable<GameLink>> Find();
```
Comment (Arthus15) [inline ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkFinder.cs]: async sufix

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkAssigner.cs (hunk 1 above)
Comment (vvillena) [reply ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkAssigner.cs]: fixed

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkFinder.cs (hunk 2 above)
Comment (vvillena) [reply ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/ILinkFinder.cs]: fixed

**Review by xaberue — APPROVED** [review]

**Pull request merged**